src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoDatabaseCustomNode.cs

```diff
@@ -32,6 +32,9 @@ protected override void InitializeChild(TreeNode parent, TreeNode child, object
 
         public override string GetNodeSubType(object smoObject, SmoQueryContext smoContext)
         {
+            // IsLedger property check is disabled on Azure databases to avoid making connections to databases 
```

Comment:
You're still calling db.IsLedger below - isn't that still resulting in querying SMO for the property and causing it to run the DB query?

Reply:
No, because it didn't exist in the [SmoTreeNodesDefinition.xml](https://github.com/microsoft/sqltoolsservice/pull/1798/files#diff-90e1717d1de73f5827918374b648cfb7f8374fa071d7f6b993303cfab4c68157) file before, IsPropertySupported always returned true with a warning.. we have had that warning in our logs.

Enabling for specific databases, returns true only for those.

Reply:
Whoops, thought this was SqlSmoObject.IsSupportedProperty!